Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/BayesQuasi.py

```diff
@@ -211,9 +211,9 @@ def PyExec(self):
             else:
                 raise ValueError('Stretched Exp ONLY works with RES file')
 
-        logger.information('Version is ' + prog)
-        logger.information(' Number of spectra = ' + str(nsam))
-        logger.information(' Erange : ' + str(erange[0]) + ' to ' + str(erange[1]))
+        logger.information('Version is %s ' % prog)
+        logger.information(' Number of spectra = %i ' % (nsam))
+        logger.information(' Erange : %f  to %f ' % (erange[0], erange[1]))
```

Comment:
If doing this type of refactoring, why not go all the way to the recommended `format()`?